src/genome_assemblies/download_summary.py

```diff
@@ -0,0 +1,48 @@
+"""Functions for downloading assembly."""
+
+import os
+from datetime import datetime
+from email.message import Message
+from urllib.request import urlretrieve
+
+
+def create_assembly_summary_url(kingdom: str) -> str:
+    return '{base_url}/{kingdom}/assembly_summary.txt'.format(
+        base_url='ftp://ftp.ncbi.nlm.nih.gov/genomes/refseq/',
+        kingdom=kingdom,
+    )
+
+
+def download_kingdom_assembly_summary(
+    kingdom: str,
+    output_filename: str,
+) -> tuple[str, Message]:
+    url = create_assembly_summary_url(kingdom)
+    return urlretrieve(url, output_filename)  # noqa: S310
+
+
+def create_output_filename(output_path: str, timestamp: bool = True) -> str:
+    pass
+
+
+def download_assembly_summaries(
+    output_path: str,
+    kingdoms: str = 'archaea,bacteria',
```

Comment:
Yeah, I want this to be usable from a command line. If it's not a string, it can be a list or a tuple. If it's a list linter will complain - for a good reason - input argument will be a mutable structure. If it's a tuple I have issue iterating it if it contains a single element, e.g.

```py
for x in ('igor'):
  print(x)

#> "i"
#> "g"
#> "o"
#> "r"
```

Wonderful! 🙄 It works properly if you define a tuple as `('igor', )` btw. But then it means if you call this function you always have to use this wonky tuple notation with an extra comma.

With `.split(',')` the return value is always a list, so its safe to use it in the for loop.

Reply:
oh yeah the comma for tuples is a little weird, that makes sense to keep it as a list of options though since this can be a CLI tool.